src/content/docs/new-relic-solutions/tutorials/add-time-picker.mdx

```diff
@@ -0,0 +1,450 @@
+---
+title: 'Add a time picker'
+metaDescription: 'Add a time picker to a sample application'
+freshnessValidatedDate: never
```

Comment:
The `freshnessValidatedDate` field is not updated in any of the files.

Reply:
I left them at never, because I think all of these will need to be revisited later.